source/blog/2015-06-18-ember-data-1-13-released.md

```diff
@@ -0,0 +1,774 @@
+
+## New Features:
+
+## Simplified Find Methods
+
+Ember Data methods on the store have grown organically over the life
+of the project. Originally, Ember Data started with `store.find(type)`
+to request all records for a type and `store.find(type, id)` to
+request a specific record. Later Ember Data added ways to issue a
+query to the server via `store.find(type, {query})` or always reload a
+record from the backend using `store.fetchById(type, id)` and
+`store.fetchAll(type)`.
+
+We realized the overloaded `store.find` method and inconsistent naming
+of other methods is confusing for both new and existing developers. As a result, we have renamed many of the existing
+store methods to make them more consistent and approachable for all
+developers.
+
+In particular, `store.find`, `store.all`, `store.getById` have been 
+deprecated and are replaced with conistently named methods. New methods follow a simple convention: If they are async and potentially go to the server, they start with `find`, and if they only get store local data wihout sideeffects they start with `peek`.
+If they return a single record they end in `Record` and if they return all the records they end in `All` .
+#### Reorganized Find Methods
+
+<table>
+  <thead>
+    <tr>
+      <th></th>
+      <th>Async from server/store</th>
+      <th>Sync from store</th>
+      <th>Query server</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><b>Single Record</b></td>
+      <td><code>findRecord(type,id)</code></td>
+      <td><code>peekRecord(type, id)</code></td>
+      <td><code>queryRecord(type, {query})</code></td>
+
+    </tr>
+
+    <tr>
+      <td><b>All Records</b></td>
+      <td><code>findAll(type)</code></td>
+      <td><code>peekAll(type)</code></td>
+      <td><code>query(type, {query})*</code></td>
+    </tr>
+ 
+  </tbody>
+</table>
+
+\* A query usually does not return all the records of a type, so doesn't end in `All` 
+
+#### query and queryRecord
+
+The final use case for the old `store.find` method was issuing queries
+to the server. This usage of `store.find(type, { query })` has been
+deprecated and replaced by a new `query` method
+on the store.
+
+```js
+store.query(type, { query });
+```
+
+In addition to `store.query` we have also added a `queryRecord` for
+issuing arbitrary queries to the backend where the expected response
+is a single record.
+
+```js
+store.queryRecord(type, { query });
+```
+
+Thanks to [@thaume](http://github.com/thaume) for his work implementing this feature.
+
+
+
+## Better caching defaults for `findAll` and `findRecord`
+
+In Ember Data beta.19 calling `store.find(type, id)` would fetch the
+fresh data from the server the first time find was called, and then every next time
+return cached data. If the user always needed fresh data, they had to
+know to call `store.fetchRecord`, and if they needed to background update
+they would have to make multiple calls and be careful about what they return from
+a route's `model:` hook.
```

Comment:
This sentence was a little confusing for me since I think it relies on a lot of deep knowledge of the context to understand what it's alluding to. Can we either try to expand on this, or cut it entirely?

Also I think we typically refer to the `model()` hook rather than the `model:` hook. I can see your ES5 showing. ;)